examples/resumption/ColdResumption_Client.cpp

```diff
@@ -44,6 +44,14 @@ class HelloSubscriber : public virtual Refcounted, public Subscriber<Payload> {
     count_++;
   }
 
+  void onError(folly::exception_wrapper) override {
```

Comment:
Not this diff but, if you call `Subscriber<Payload>::onSubscribe(sub)` inside of `HelloSubscriber::onSubscribe(sub)`, then you won't need these overloads, and you'll be able to access `sub` via `Subscriber<Payload>::subscription()`.